### DISCUSSION TITLE
Support for AVIF?

### DISCUSSION THREAD

**Discussion by snidelytoo** [python-pillow/Pillow#8337] [Ideas]
AVIF appears to be a new open-source format with browser support.  Is there any plan to add this format to the list that Pillow supports?  I've been using Pillow for making thumbnails for a while, and plan to try making collages soon.

**Comment by Yay295** [comment]
#5201

**Reply by snidelytoo** [reply]
Thank you!   Missed that in my first time browsing this project.
